### DISCUSSION TITLE
Are there any shortcuts to quickly accept copilot's code

### DISCUSSION THREAD

**Discussion by FengZi-lv** [community/community#55970] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Are there any shortcuts to quickly accept copilot's code
![image](https://github.com/community/community/assets/59948673/d30d13ed-dfa3-4fc2-8c81-a4412490c20f)

thanks

**Comment by Akash1134** [comment]
Hi @FengZi-lv , To accept an inline suggestion you can press Tab on your keyboard. If you're looking for something else, please explain in more detail. Thanks!

**Reply by FengZi-lv** [reply]
Yes,copilot can use Tab to accept suggestions, but copilot x cannot use Tab to accept suggestions

**Reply by FengZi-lv** [reply, marked as answer]
copilot x seems to be able to take advice using Esc
